### PR TITLE
Feat: add full hypixel autoblock

### PR DESCRIPTION
**Pull request by localpthebest**
lasted 2 hours straight so shouldnt silent 👍

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/killaura/features/AutoBlock.kt [hunk 1]
```diff
@@ -95,13 +93,6 @@ object AutoBlock : ToggleableConfigurable(ModuleKillAura, "AutoBlocking", false)
...
-                network.sendPacket(Full(player.x, player.y, player.z, player.yaw, player.pitch, player.isOnGround))
-                network.sendPacket(
-                    PlayerActionC2SPacket(
-                        PlayerActionC2SPacket.Action.STOP_DESTROY_BLOCK,
-                        player.blockPos,
-                        player.horizontalFacing.opposite)
-                )
                 true
             }
             else -> false
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/killaura/features/AutoBlock.kt]: Function can be replaced with a simple return tickOn == 0

**Comment by levzzz5154** [comment]
shit code stop coding bro fr please

**Comment by larryngton** [comment]
> shit code stop coding bro fr please

:fire:

**Comment by ghost** [comment]
This is skidded from local p's brain cells

**Comment by localpthebest** [comment]
> shit code stop coding bro fr please

ive yet to see you commit shit.
come back when youve made ur own prs

**Comment by localpthebest** [comment]
> This is skidded from local p's brain cells

frfr

**Comment by EclipsesDev** [comment]
you sure this work? have u check hypixel replay

**Pull request merged**